ci/dra_common.sh

```diff
@@ -0,0 +1,33 @@
+function info {
+    echo "INFO: $1"
+}
+
+function error {
+    echo "ERROR: $1"
+}
+
+function save_docker_tarballs {
+    ver=$1
```

Comment:
In this function, we receive the value of a global variable and assign it to a local one, instead of directly accessing the global variable.